modules/file_upload/cms_file_upload.e

```diff
@@ -1,248 +0,0 @@
-		do
-			if a_block_id.same_string ("uploader") then
-				-- create menu_block.make ({CMS_MENU_SYSTEM}.primary_menu)
-				-- a_response.add_block (menu_block, "page_top")
-			end
-		end
-
-	menu_system_alter (a_menu_system: CMS_MENU_SYSTEM; a_response: CMS_RESPONSE)
-		local
-			link: CMS_LOCAL_LINK
-			second_link: CMS_LOCAL_LINK
-		do
-			-- login in demo did somehow not work
-			-- if a_response.has_permission ("upload files") then
-				create link.make ("Upload", "upload/")
-				a_menu_system.primary_menu.extend (link)
-			-- end
-		end
-
-feature -- Configuration		
-
-	document_root: PATH
-			-- Document root to look for files or directories
-		once
-			Result := execution_environment.current_working_path.extended ("site")
-		end
-
-	files_root: PATH
-			-- Uploaded files will be stored in `files_root' folder
-		local
-			tmp: PATH
-		once
-			tmp := document_root.extended ("files")
-			Result := tmp.extended ("uploaded_files")
-		end
-
-feature -- Handler
-
-	execute_not_found_handler (uri: READABLE_STRING_8; req: WSF_REQUEST; res: WSF_RESPONSE)
-			-- `uri' is not found, redirect to default page
-		do
-			res.redirect_now_with_content (req.script_url ("/"), uri + ": not found. %N Redirectioin to" + req.script_url ("/"), "text/html")
-		end
-
-	execute_upload (req: WSF_REQUEST; res: WSF_RESPONSE)
-		local
-			body: STRING_8
-		do
-			-- create body
-			create body.make_empty
-			body.append ("<h1> EWF: Upload files </h1>%N")
-			body.append ("<p>Please choose some file(s) to upload.</p>")
-
-			-- create form to choose files and upload them
-			body.append ("<form action=%"" + req.script_url ("/upload/") + "%" enctype=%"multipart/form-data%" method=%"POST%"> %N")
-			body.append ("<input name=%"file-name[]%" type=%"file%" multiple> %N")
-			body.append ("<button type=submit>Upload</button>%N")
-			body.append ("</form>%N")
-
-			-- put the body to the response
-			res.put_string (body)
-
-			show_and_store_files (req, res)
-
-		end
-
-	show_and_store_files (req: WSF_REQUEST; res: WSF_RESPONSE)
-			-- show all uploaded files
-		local
-			file_path: PATH
-			file_name: STRING_8
-			stored: BOOLEAN
-			file_system_handler: WSF_FILE_SYSTEM_HANDLER
-			file_system_upload_handler: CMS_FILE_UPLOAD_FILE_SYSTEM_HANDLER
-			uploaded_file: CMS_FILE_UPLOAD_FILE
-		do
-			-- if has uploaded files, then store them
-			if req.has_uploaded_file then
-				across
-					req.uploaded_files as uf
-				loop
-					file_name := uf.item.safe_filename
-
-					-- check if file is already in folder
-					if not files_root.has_extension (file_name) then
-						file_path := files_root.extended (file_name)
-
-						-- move file to path
-						stored := uf.item.move_to(file_path.name)
-					end
-				end
-			end
-
-			-- create file_system_handler and show the uploaded files
-			create file_system_handler.make_with_path (document_root)
-			file_system_handler.enable_index
-			file_system_handler.process_index ("/uploaded_files", files_root, req, res)
```

Comment:
here this is bad too ... since file system handler will produce a full html
so you ends up with ... invalid html content.
In addition, process_index will try to send expected http header .. but since previous in the code, you already response with html .. you can not try to send new header like that.

I guess in the console/terminal/tty you have warning related to that.!